### PR TITLE
refactor: remove duplicate code for incrementing counters

### PR DESCRIPTION
**Pull request by 56quarters**
Delegate to count method and use glob import for metric traits

### CODE REVIEW

File: src/server/metrics.rs [hunk 1]
```diff
@@ -2,9 +2,8 @@ use std::net::UdpSocket;
 use std::time::Instant;
 
 use actix_web::{error::ErrorInternalServerError, web::Data, Error, HttpRequest};
-use cadence::{
-    BufferedUdpMetricSink, Counted, Metric, NopMetricSink, QueuingMetricSink, StatsdClient, Timed,
-};
+use cadence::prelude::*;
```
Comment (jrconlin) [inline src/server/metrics.rs:5]: We prefer not to do global includes like this for a number of reasons (mostly, it's easier for folk to find the source of a given method or structure if we note where it comes from, avoids some accidental collisions or usages, etc.).
Could you list the structs and methods you're actually using from inside of `cadence::prelude`?

**Review by jrconlin — CHANGES_REQUESTED** [review]
Hi!
Thanks for this. It looks good, with one style nit.

File: src/server/metrics.rs (hunk 1 above)
Comment (56quarters) [reply src/server/metrics.rs:5]: Sure, will do.

**Review by jrconlin — APPROVED** [review]

**Comment by 56quarters** [comment]
Working on fixing commit signing. Sorry about that.

**Comment by jrconlin** [comment]
Sigh...
For security reasons, we need to have every commit associated with the PR signed. The original commit was not. 

My apologies, but could you either rebase this PR to a signed commit or resubmit with all the commits signed? If this is too annoying, let me know and I'll resubmit for you, (with appropriate credit).

Sorry for the hassle.

**Comment by jrconlin** [comment]
Oh, man, NO NEED to apologize for us making things harder.

Seriously, thank you for taking the time to submit a PR. It means a lot.

**Review by jrconlin — APPROVED** [review]

**Comment by jrconlin** [comment]
@56quarters Just in case you don't want to deal with github being annoying, I resubmitted your patch in #983. 

Feel free to review, comment, and point out what I may have gotten wrong.

**Comment by 56quarters** [comment]
Closed in favor of #983

**Pull request closed without merging**